### PR TITLE
de_DE | Updates

### PR DESCRIPTION
**Pull request by Vrontis**
Wasn't quite sure with some of the ratelimit/mod quotes though

### CODE REVIEW

File: src/main/resources/assets/languages/de_DE.json [hunk 1]
```diff
@@ -69,7 +69,11 @@
...
+    "boom_quotes": ["Offenbar hat Megumin unser Schloss in die Luft gejagt...", "Uh-oh, offenbar hat unser Meister ein paar Nullen und Einsen auf dem Boden vergessen :<", "W-Warte, was ist gerade passiert?", "I-Ich denke hier brennt's... Du solltest vielleicht meinen Meister bitten, sich das anzuschauen.", "Ich habe Explosionsmagie gemeistert, siehst du?", "Vielleicht ist hier einfach etwas schiefgelaufen, aber, u-uh, Ich kann es nicht richtigstellen!", "U-Uhh.. Was wolltest du?"],
+    "ratelimit_quotes": ["Woah... you're calling me a bit too fast... I might get dizzy!", "Sei nicht gierig!", "Y-You're calling me so fast that I'm getting dizzy...", "Wart mal!", "Warte nur einen kleeiinen Augenblick länger uwu", "Schaut so aus, als würden wir noch ein Bußgeld bekommen, wenn wir so schnell weitermachen!", "Ich will das machen... aber warte noch einen Moment bitte.", "Hey, wart mal, Ich bin mit meiner Pause noch nicht fertig!", "Kannst du ein bisschen langsamer machen?"],
```
Comment (MrLar) [inline src/main/resources/assets/languages/de_DE.json]: `Woah... you're calling me a bit too fast... I might get dizzy!` can be translated to `Woah... Du rufts mich viel zu schnell... Mir wird ganz schwindlig.`
`Y-You're calling me so fast that I'm getting dizzy...` can be translated to `D-Du rufst much viel zu schnell.. mir ist schwindlig`

File: src/main/resources/assets/languages/de_DE.json [hunk 2]
```diff
@@ -167,6 +171,10 @@
       "info": "%1$sDu kannst deine Serversprache (sofern anwendbar) mithilfe von `~>opts language set <language code>` ändern. Deine eigene Sprache kannst du mithilfe von `~>profile lang <language code>` ändern.\n**Verfügbare Sprache:** `%2$s`\nSollte deine Sprache hier nicht mitaufgeführt sein und du hast Lust zu helfen, kannst du gerne dem Support-Server beitreten und bei der Übersetzung helfen!"
     },
     "music_general": {
+      "music_error": {
+        "timeout": "%1$sNicht in der Lage zu %2$s zu verbinden: connection timed out. Versuche die Serverregion zu ändern und versuch den Befehl erneut.",
```
Comment (MrLar) [inline src/main/resources/assets/languages/de_DE.json]: translate connection timed out to Verbindugsfehler

File: src/main/resources/assets/languages/de_DE.json [hunk 3]
```diff
@@ -167,6 +171,10 @@
...
+        "unknown": "%1$sNicht in der Lage zu %2$s zu verbinden: unknown error"
```
Comment (MrLar) [inline src/main/resources/assets/languages/de_DE.json]: Translate unknown error to Unbekannter Fehler

File: src/main/resources/assets/languages/de_DE.json (hunk 1 above)
Comment (MrLar) [reply src/main/resources/assets/languages/de_DE.json]: Typo in "kleinen" (kleeiinen)

Comment (Vrontis) [reply src/main/resources/assets/languages/de_DE.json]: On purpose original string was "tiiiiny" so changed it on purpose to that.
For the other two strings I had the same idea, though its meaning is somehow weird. Would take it though, if you think it's good enough.

Comment (MrLar) [reply src/main/resources/assets/languages/de_DE.json]: The first two strings work that way translated there is no real good way to translate them but it's better than leaving them in english. And for the longer tiny you could do "kleiiinen" similiar to how english does it but basically breaking the word entirely is kinda weird.

File: src/main/resources/assets/languages/de_DE.json (hunk 2 above)
Comment (MrLar) [reply src/main/resources/assets/languages/de_DE.json]: *Verbindungsfehler (im good at german trust me :mmlol:)

File: src/main/resources/assets/languages/de_DE.json (hunk 1 above)
Comment (Vrontis) [reply src/main/resources/assets/languages/de_DE.json]: Ok, guess I'll take this that way~

File: src/main/resources/assets/languages/de_DE.json (hunk 2 above)
Comment (natanbc) [reply src/main/resources/assets/languages/de_DE.json]: why does german always sound like you're screaming?
english: "Connection timed out"
german: "VERBINDUNGSFEHLER"

Comment (MrLar) [reply src/main/resources/assets/languages/de_DE.json]: Why are you meming on an actual review thats a better question

Comment (Vrontis) [reply src/main/resources/assets/languages/de_DE.json]: Wouldn't connection timed out -> Zeitüberschreitung be better?

Comment (MrLar) [reply src/main/resources/assets/languages/de_DE.json]: Would also work but wouldnt be as understandable by people who don't know much about package flow

**Pull request merged**